source/components/organisms/Step/StepFooter/StepFooter.tsx

```diff
@@ -203,6 +207,7 @@ StepFooter.propTypes = {
     start: PropTypes.func,
     /** action to return to main form */
```

Comment:
brus

Reply:
Again, maybe we can limit comments to the PR? At least for minor stylistic things, of course if someone spots a larger flaw or bug, then sure.

Reply:
Fixed.